_pages/gift.md

```diff
@@ -0,0 +1,44 @@
+---
+mediawiki: GIFT
```

Comment:
This tag is intended for pages migrated from the old mediawiki that haven't been reviewed yet. It'll create a warning like:

> The content of this page has not been vetted since shifting away from MediaWiki. If you’d like to help, check out the [how to help guide](https://imagej.net/events/wiki-grand-opening/how-to-help)!

I suggest removing it.

```suggestion
```